crates/mysticeti-core/src/consensus/base_committer.rs

```diff
@@ -160,22 +169,22 @@ impl BaseCommitter {
 
     /// Check whether the specified leader has enough blames (that is, 4f+1 non-supports) to be
     /// directly skipped.
-    fn enough_leader_blame(&self, decision_round: RoundNumber, leader: AuthorityIndex) -> bool {
-        let decision_blocks = self.block_store.get_blocks_by_round(decision_round);
+    fn enough_leader_blame(&self, voting_round: RoundNumber, leader: AuthorityIndex) -> bool {
```

Comment:
ok I think I confused you. let's rename voting_round -> booster round. This here must also be the decision round.